apps/sample-next-auth-app/README.md

```diff
@@ -1,60 +1,60 @@
 # Xata + Next.js + Auth.js
 
-Sample app using Next-Auth for managing OAuth login with the Xata adapter.
+This is a simple dashboard built using Next 13 with Auth.js to manage OAuth & the official [Xata adapter](https://authjs.dev/reference/adapter/xata) to capture user storage, sessions, and more!
 
-> ⚠️ This example uses the **App Routing**, if using `/pages`, go for **Pages Routing**.
+> ⚠️ This example uses the Next 13 `app router`. If you're using `page routing` you can find an example [here](https://github.com/xataio/examples/tree/main/apps/sample-next-auth-pages).
 
 ## Features ⚡️
 
 - React Server Components
-- Next-Auth API Routes.
-- GitHub as OAuth Provider with Xata official adapter.
+- Auth.js Route Handlers
+- GitHub as OAuth Provider
+- User sessions, storage, and more in your database using the Xata adapter
 
 ### Cloning the Project
 
+We'll be using [degit](https://github.com/Rich-Harris/degit) to clone the example locally, without any of the git history
+
 ```bash
-# create a new project in my-awesome-app
-npx degit xataio/examples/apps/sample-next-auth-app my-awesome-app
+npx degit xataio/examples/apps/sample-next-auth-app your/project/name
 ```
 
 ## Initialize your Database
 
-In order to connect to a workspace, a `XATA_API_TOKEN` and a database URL. One of way of setting it up is running the [Xata CLI](https://xata.io/docs/getting-started/cli).
+First, make sure you've globally installed the [Xata CLI](https://xata.io/docs/getting-started/cli).
```

Comment:
I'd add a note that even if it's already installed, it needs to be up-to-date because the SDK is set to "latest", so an older version of CLI may generate incompatible types and fail silently.

Reply:
Very fair, I'll make sure there's a note added. Great callout.